pkg/provider/operators.go

```diff
@@ -37,6 +34,7 @@ type Operator struct {
 	Org              string                            `yaml:"org" json:"org"`
 	Version          string                            `yaml:"version" json:"version"`
 	Channel          string                            `yaml:"channel" json:"channel"`
+	NameOnly         string                            `yaml:"NameOnly" json:"NameOnly"`
```

Comment:
Does camel-case work in the `json` identifier?

Reply:
fixed